DataFormats/Math/interface/Graph.h

```diff
@@ -407,25 +401,21 @@ void graph<N,E>::clear()
 }
 
 template<class N, class E>
-void graph<N,E>::dump_graph() const
+void Graph<N,E>::dump_graph() const
 {
-  //  std::cout << adjl_ << std::endl;
-  /*
-    std::cout << "Nodes and their indices:" << std::endl;
-    typename indexer_type::const_iterator it = indexer_.begin();
-    for (; it != indexer_.end(); ++it) {
-    std::cout << ' ' << it->first << ' ' << it->second << std::endl;
-    }
-  */   
+  std::cout << "Nodes and their indices:" << std::endl;
+  for( auto const & it : indexer_ ) {
+    std::cout << ' ' << it.first << ' ' << it.second << std::endl;
```

Comment:
there should probably be a MessageLogger-aware version, otherwise this interface is unusable in multithreaded mode.